R/pkg/R/DataFrame.R

```diff
@@ -354,6 +354,24 @@ setMethod("colnames<-",
             dataFrame(sdf)
           })
 
+specialtypeshandle <- function(type) {
```

Comment:
maybe this should go into types.R?
Could you add some documentation comment what this is doing and why?